pioneer/packages/joy-proposals/src/Proposal/ProposalDetails.tsx

```diff
@@ -106,32 +135,41 @@ function ProposalDetails ({
   const extendedStatus = getExtendedStatus(proposal, bestNumber);
   const isVotingPeriod = extendedStatus.periodStatus === 'Voting period';
   return (
-    <Container className="Proposal">
+    <div className="Proposal">
       <Details proposal={proposal} extendedStatus={extendedStatus} proposerLink={ true }/>
-      <Body
-        type={ proposal.type }
-        title={ proposal.title }
-        description={ proposal.description }
-        params={ proposal.details }
-        iAmProposer={ iAmProposer }
-        proposalId={ proposalId }
-        proposerId={ proposal.proposerId }
-        isCancellable={ isVotingPeriod }
-        cancellationFee={ proposal.cancellationFee }
-      />
-      { iAmCouncilMember && (
```

Comment:
Based on https://github.com/Joystream/joystream/issues/365 there were two models of access that were considered:

> (...) Write access as one among two modes, either open, in which case any member can write, or closed, in which case only proposer and council can initially write, but any third party can be invited to write as well.

I think only the _open_ model was actually implemented, so now any member can post in a discussion. Such posts (from members outside of council) are accepted by the runtime, so I wasn't trying to prevent them from the UI either.